Source/Core/Core/IOS/IPC.cpp

```diff
@@ -412,6 +414,11 @@ static void SDIO_EventNotify_CPUThread(u64 userdata, s64 cycles_late)
     device->EventNotify();
 }
 
+u32 GetVersion()
+{
+  return static_cast<u32>(s_active_title_id);
```

Comment:
It's a title ID (u64), where the first 32 bytes are used for the title type (it will always be 00000001 for system titles like IOS), and the lower 32 bytes are used for the IOS version (e.g. 00000009 for IOS9). I'll add a comment to clarify this.